diff -ur nas-1.2p5/lib/audio/Alibint.c nas-1.2p5.fixed/lib/audio/Alibint.c
--- nas-1.2p5/lib/audio/Alibint.c	Thu Dec 28 20:42:47 1995
+++ nas-1.2p5.fixed/lib/audio/Alibint.c	Sun Dec 21 04:08:21 1997
@@ -1324,7 +1324,7 @@
 static char *_SysErrorMsg (n)
     int n;
 {
-#if !defined(__FreeBSD__)
+#if !defined(__FreeBSD__) && !defined(__linux__)
     extern char *sys_errlist[];
 #endif
     extern int sys_nerr;
