diff -uNr nas-1.2p5.orig/clients/audio/audial/Imakefile nas-1.2p5/clients/audio/audial/Imakefile
--- nas-1.2p5.orig/clients/audio/audial/Imakefile	Tue Sep 17 00:39:18 1996
+++ nas-1.2p5/clients/audio/audial/Imakefile	Tue Sep 17 01:11:14 1996
@@ -28,7 +28,7 @@
   DIR = audial
 
 #if defined(HPArchitecture) || defined(UltrixArchitecture) || defined(SGIArchitecture) || defined(SystemV4) || defined(i386SVR3Architecture)
-     USLEEPDEFS = -DNEEDUSLEEP
+     /*USLEEPDEFS = -DNEEDUSLEEP*/
 #endif
         DEFINES = $(USLEEPDEFS) 
 
